swim/events.go

```diff
@@ -140,3 +157,31 @@ type ChecksumComputeEvent struct {
 	Duration time.Duration `json:"duration"`
 	Checksum uint32        `json:"checksum"`
 }
+
+// A ChangesCalculatedEvent is sent when the disseminator generated the list of changes to send in a ping or its response
+type ChangesCalculatedEvent struct {
+	Changes []Change `json:"changes"`
```

Comment:
Don't think so, but all the events in this package had them defined. Didn't really look into why, but kept it consistent here.

Happy to not add them to these events, but rather not remove them from the existing ones since I don't really know why they are on there.